CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 1.0.0
```

Comment:
This should be pre-release. Do you reckon?

Reply:
I will ask for clarification on Slack